apps/designer/app/designer/features/style-panel/shared/use-style-data.ts

```diff
@@ -174,27 +174,31 @@ export const useStyleData = ({
       updates.push({ operation: "delete", property });
     };
 
-    const publish = () => {
+    const publish = (options = { isEphemeral: false }) => {
       if (!updates.length) {
         return;
       }
-      publishUpdates("update", updates);
-      setBreakpointStyle((prevStyle) => {
-        const nextStyle = updates.reduce(
-          (reduceStyle, update) => {
-            if (update.operation === "delete") {
-              delete reduceStyle[update.property];
-            }
-            if (update.operation === "set") {
-              reduceStyle[update.property] = update.value;
-            }
-            return reduceStyle;
-          },
-          { ...prevStyle }
-        );
-        updates = [];
-        return nextStyle;
-      });
+      const type = options.isEphemeral ? "preview" : "update";
+      publishUpdates(type, updates);
+
+      if (options.isEphemeral === false) {
+        setBreakpointStyle((prevStyle) => {
```

Comment:
we renamed breakpoint style to local style, maybe its not merged yet, @TrySound

Reply:
Probably not merged. I didn't change this code, only wrapped it into an `if`